src/Middleware/Session/src/DistributedSession.cs

```diff
@@ -35,6 +38,23 @@ public class DistributedSession : ISession
         private string _sessionId;
         private byte[] _sessionIdBytes;
 
+        /// <summary>
+        /// Initializes a new instance of <see cref="DistributedSession"/>.
+        /// </summary>
+        /// <param name="cache">The <see cref="IDistributedCache"/> used to store the session data.</param>
+        /// <param name="sessionKey">A unique key used to lookup the session.</param>
+        /// <param name="idleTimeout">How long the session can be inactive (e.g. not accessed) before it will be expire.</param>
```

Comment:
```suggestion
        /// <param name="idleTimeout">How long the session can be inactive (e.g. not accessed) before it will be expired.</param>
```

Reply:
Good catch. I think I'll go with the following:

```suggestion
        /// <param name="idleTimeout">How long the session can be inactive (e.g. not accessed) before it will expire.</param>
```